solidity/utils/stats_migration.js

```diff
@@ -0,0 +1,335 @@
+const fs = require("fs");
+const path = require("path");
+const Web3 = require("web3");
+
+const SOURCE_NODE   = process.argv[2];
+const TARGET_NODE   = process.argv[3];
+const STORE_ADDRESS = process.argv[4];
+const STORE_BLOCK   = process.argv[5];
+const PRIVATE_KEY   = process.argv[6];
+const TEST_MODE     = process.argv[7];
+
+const MIN_GAS_LIMIT = 100000;
+
+const CFG_FILE_NAME = "stats_migration.json";
```

Comment:
It looks like this file wasn't Prettified. Why?

Reply:
Because I haven't executed `npm run format`.

Reply:
You can also configure your vscode to do this automatically whenever you save a file.

Reply:
It's still not prettified

Reply:
There are some 50 files unprettified. Which ones do you want to prettify under this PR?

Reply:
Only files that were added/affected by this PR (like we do we any other refactoring)

Reply:
Done.